### PR TITLE
Adds documentation about the metrics shared

### PR DESCRIPTION
**Pull request by josemigallas**
JIRA: https://issues.jboss.org/browse/AEROGEAR-1930

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,4 +32,56 @@ Just drop the jar into the _providers_ subdirectory of your KeyCloak installatio
...
+A variety of JVM metrics are provided
+
+### Generic events
+Every single internal Keycloak event is being shared through the endpoint, with the descriptions `Generic KeyCloak User event` or `Generic KeyCloak Admin event`. Most of these events are not likely useful for the majority users but are provided for good measure. A complete list of the events can be found at [Keycloak documentation](http://www.keycloak.org/docs-api/3.2/javadocs/org/keycloak/events/EventType.html).
+
+### Featured events
+There are however a few events that are particularly more useful from a mobile app perspective. These events have been overriden by the SPI and are described more thoroughly below.
+
+##### keycloak_logins
+This counter counts every login performed by a standard user. It also distinguishes logins by the utilised identity provider by means of the label **provider**.
```
Comment (pb82) [inline README.md]: What's a standard user? Do you mean non-admin user?

File: README.md [hunk 2]
```diff
@@ -32,4 +32,56 @@ Just drop the jar into the _providers_ subdirectory of your KeyCloak installatio
...
+
+```c
+# HELP keycloak_logins Total successful logins
+# TYPE keycloak_logins gauge
+keycloak_logins{realm="test",provider="keycloak",} 3.0
+keycloak_logins{realm="test",provider="github",} 2.0
+```
+
+##### keycloak_failed_login_attempts
+This counter counts every login, performed by a standard user, that fails due to a nonexistent user or a wrong password. It also distinguishes logins by the identity provider used by means of the label **provider**.
```
Comment (pb82) [inline README.md]: We could even log the failure reason (wrong password, unknown user) in a label. I think the metrics are good for now but we have that option.

File: README.md (hunk 1 above)
Comment (josemigallas) [reply README.md]: Yes, should it say _non-admin_ instead?

File: README.md (hunk 2 above)
Comment (josemigallas) [reply README.md]: Yes good suggestion, I will update it

Comment (pb82) [reply README.md]: No need to log that for the first version, but it's good to know that we can.

File: README.md (hunk 1 above)
Comment (pb82) [reply README.md]: Yes, i think that would be better

**Comment by josemigallas** [comment]
@pb82 both comments addressed

**Comment by psturc** [comment]
LGTM, I'll just verify the changes locally...

**Pull request merged**